packages/ses/src/make-evaluate.js

```diff
@@ -43,6 +49,10 @@ export const makeEvaluate = context => {
     globalLexicalConstants,
     'globalLexicals',
   );
+  const moduleLexicalOptimizer = buildOptimizer(
+    moduleLexicalConstants,
+    'moduleLexicals',
+  );
```

Comment:
Can this ever actually happen? I mean this is probably the safest way, and doesn't cost much.

Reply:
Not in any correct arrangement. Any potential overlap would need to be censored by a transform.